src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt

```diff
@@ -58,6 +60,7 @@ class ChristmasEventPlugin : JavaPlugin() {
         registerCommands()
         registerEvents()
         registerPacketAPI()
+        refreshAccessToken()
```

Comment:
Rename this to `handleDonations()` and put all init donation code inside.

```suggestion
        handleDonations()
```